lib/server/routes/buckets.js

```diff
@@ -1396,6 +1399,111 @@ BucketsRouter.prototype.getStorageUsage = function (req, res) {
   });
 };
 
+BucketsRouter.prototype.startUpload = function (req, res, next) {
+  const { Bucket, Upload, Contact } = this.storage.models;
+
+  const { uploads } = req.body;
+
+  if (!uploads) {
+    return next(new errors.BadRequestError('Missing uploads'));
+  }
+
+  if (!Array.isArray(uploads)) {
+    return next(new errors.BadRequestError('Uploads is not an array'));
+  }
+
+  uploads.forEach(({ index, size }) => {
+    if (typeof size !== 'number' || size < 0) {
+      return next(new errors.BadRequestError('Invalid size'));
```

Comment:
Same as #52, be careful as this return will not end the execution nor the `foreach` neither `startUpload` function

Reply:
Missed this completely, fixed